presets/preset_ExtremeShuffle.yaml

```diff
@@ -169,7 +168,8 @@ RandomPartnerFP  : 3
 RandomStarpowerSP: 3
 
 # Misc Gameplay Randomization
-ShuffleBlocks: true
+ShuffleBlocks   : true
+RandomizePuzzles: false
```

Comment:
Should we randomize puzzles in the extreme preset?

Reply:
Sure, sounds like that would be in the spirit of extreme